creusot-contracts/src/logic.rs

```diff
@@ -5,8 +5,9 @@ mod fset;
 mod int;
 mod mapping;
 mod ops;
-mod ord;
+pub mod ord;
 mod seq;
+mod seq2;
```

Comment:
Sorry, I was also experimenting with a version of `seq` implemented directly in Rust instead of using Why3 so that it would be monomorphized, and forgot to remove it from the commit